config/application.rb

```diff
@@ -33,10 +33,6 @@ class Application < Rails::Application
       event.payload.except(:params, :headers)
     end
 
-    # Refuse to run if migraitons are pending. Migrations should be run out
-    # of band from launching the app
-    config.active_record.migration_error = :page_load
```

Comment:
I can't find any other options for this setting, is this check still run when the app boots?

If not, maybe we could update `DatabaseHealthChecker` to check for migrations as well, so we can do health checks at `api/health/database`?

Reply:
I see now that the change was made [here](https://github.com/18F/identity-idp/pull/3435/files) and it also added a check for migrations on deploy, which should be good?

Reply:
Ah yup! Perfect, thanks for confirming